### ISSUE TITLE
Can't remove requests from 1.9.7

### ISSUE THREAD

**Issue by damiantje99** [tidusjar/Ombi#862]
#### Ombi Version:

V 1.10.1

#### Update Branch:

Stable

#### Operating System:

Linux ubuntu

#### Applicable Logs (from `/logs/` directory or the Admin page):

```

System.ArgumentException: Cannot build uri when url segment parameter 'imdbid' value is null. Parameter name: request at RestSharp.RestClient.BuildUri (IRestRequest request) <0x41d29820 + 0x00233> in :0 at RestSharp.RestClient.ConfigureHttp (IRestRequest request, IHttp http) <0x41d25360 + 0x0044b> in :0 at RestSharp.RestClient.Execute (IRestRequest request, System.String httpMethod, System.Func`3 getResponse) <0x41d215f0 + 0x0009b> in :0

```

#### Problem Description:

After I updated PlexRequest from 1.9.7, I can't remove or approve the requests that were made in that version (Movies and Shows). New requests, that are done in the current version, do work fine. I can approve or remove them.

**Comment by tidusjar** [comment]
That's strange,

Can you change your admin password and send me your database so I can take a look? (Also send me the new admin password) tidusjar@gmail.com

Thanks

**Comment by damiantje99** [comment]
I've sent you an email.

**Comment by tidusjar** [comment]
Looking at your database it seems that all of the requests are slightly corrupt (missing data). Not sure how this has happened and I've never seen it before, but I have made a code change that fixes this and will put the missing data back into the object to stop this from happening again.

You are welcome to switch to the Early Access Preview branch where these changes are.

https://ci.appveyor.com/project/tidusjar/requestplex/build/artifacts

Since the Early Access Preview branch is version 2.0 it will require a fresh install. Install into a fresh location and copy over your `PlexRequests.sqlite` file.

**Comment by damiantje99** [comment]
I want to try this, but in the new version, there is no file called PlexRequests.sqlite. Do I need to rename this file before copying it?

**Comment by tidusjar** [comment]
The PlexRequests.sqlite is generated when the app starts up! So just copy over the file you emailed me into the new install directory and you are good to go :)

**Comment by damiantje99** [comment]
Thanks, will try that. I'll let you know.

**Comment by tidusjar** [comment]
Excellent! If you have any issues let me know 😄

**Comment by damiantje99** [comment]
Your awesome, it works! Also, i've had some trouble that PlexRequest didn't index the shows and movies from Plex. In this version it works, don't know if it was my buggy PlexRequest of that you fixed it in 2.0. Anyways, thanks for the help! Keep on rock'n.

**Comment by tidusjar** [comment]
Excellent! I have made a few improvements all around the app :)
